docker/1.4.1/mms-entrypoint.py

```diff
@@ -6,7 +7,7 @@
 if sys.argv[1] == 'serve':
     serving.main()
 else:
-    subprocess.check_call(sys.argv[1:])
+    subprocess.check_call(shlex.split(' '.join(sys.argv[1:])))
```

Comment:
Does this run with a command like:
﻿﻿﻿﻿﻿docker run -itd -v /home/ubuntu/src/container_tests:/test --name inference_mxnet -p 80:8080  -p 8081:8081 841569659894.dkr.ecr.us-east-1.amazonaws.com/beta-mxnet-inference:1.4.1-py3-cpu-build mxnet-model-server --start '--mms-config /home/model-server/config.properties' '--models  squeezenet=https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model'

Reply:
yep!